test_runner/sql_regress/expected/neon-spgist.out

```diff
@@ -0,0 +1,14 @@
+create table spgist_point_tbl(id int4, p point);
```

Comment:
Can you add a comment as to what we're testing / why that's interesting / how we found about the need to cover that part of the code?